### ISSUE TITLE
Add SVD implementation

### ISSUE THREAD

**Issue by jimfleming** [fomorians/pyoneer#30]

**Comment by djsaunde** [comment]
https://www.tensorflow.org/api_docs/python/tf/linalg/svd?

**Comment by jimfleming** [comment]
I'm not sure if this is equivalent. TF's svd may take the analytical approach. @velocirabbit's implementation used gradient descent so it would be fast with very large matrices.
